src/tests/types.ts

```diff
@@ -387,6 +391,12 @@ export type TestRunIdResult = {
 };
 
 export type TestResult = {
+  summary: Omit<TestResultRaw['summary'], 'testSetupTime'>;
+  tests: Omit<ApexTestResultData, 'isTestSetup'>[];
+  codecoverage?: CodeCoverageResult[];
+};
+
+export type TestResultRaw = {
```

Comment:
@CristiCanizales section tests should be of type `ApexTestResultDataRaw` per spec